### ISSUE TITLE
Expose active accounts count and active generalized accounts count

### ISSUE THREAD

**Issue by sborrazas** [aeternity/ae_mdw#751]
refs #124 

@marc0olo what would be considered an "active" account?

**Comment by marc0olo** [comment]
> @marc0olo what would be considered an "active" account?

IMO accounts that have been seen on the network, meaning not returning a 404 from node 😅 but it's kind of debatable in that sense because you can send AE or AEX-9 tokens to accounts which will then never turn into "active accounts".

I think that's sth. we have to agree upon. what do you think @thepiwo ?

maybe we could even distinguish between those two types:
- passive in existence (never performed a tx, but seen on the network)
- active in existence (performed at least one transaction, including outgoing tx via `PayingForTx`)

**Comment by thepiwo** [comment]
@marc0olo sounds good

**Comment by marc0olo** [comment]
cool :)

I think for generalized accounts count we more or less have that already to some extent by calling https://mainnet.aeternity.io/mdw/v2/txs/count?type=ga_attach

but I am actually not sure if this is a reliable number because the `GaAttachTx` could also revert, right? 🤔 

anyway, I would consider every valid and successful `GaAttachTx` as active GA. I see no reason to declare them as "passive".

just thinking about this topic since we aim to display the amount of GA contracts currently on explorer dashboard in the contracts section.

**Comment by thepiwo** [comment]
@sborrazas do you have any input on that?

I agree, `GaAttachTx` could revert, but would probably still be included in the count, maybe we can make a `count_ok` or similar.

**Comment by sborrazas** [comment]
Counting GA accounts that way would be an easy feature to incorporate. Counting total node accounts wouldn't be that easy, though

**Comment by thepiwo** [comment]
For GA this is already fixed, for normal accounts its expensive to compute similar to #755

Maybe we can include fixing count per transaction type fix for successful within this. @marc0olo would that be good?

**Comment by marc0olo** [comment]
I think it needs to be a separate attribute in the stats endpoint if it's about the GA contract count.

for counting `ga_attach` I would still expect to have a count for all txs, including the failed/reverted txs.

opinions?

**Comment by sborrazas** [comment]
@marc0olo this information is already exposed in the txs count endpoint filtered by tx_type: https://mainnet.aeternity.io/mdw/v2/txs/count?type=ga_attach

**Comment by marc0olo** [comment]
yes, I just want to make sure we don't replace that count somehow.

> Maybe we can include fixing count per transaction type fix for successful within this. @marc0olo would that be good?

not sure if I understand this question correctly. I think if we add the correct count for GA contracts in the stats endpoint within this I'd be happy =)
